### PR TITLE
Replace PUBLIC_IP_ICON with VPN_ICON if VPN is up

### PR DESCRIPTION
**Pull request by lemaral**
Replace PUBLIC_IP_ICON with VPN_ICON if $POWERLEVEL9K_PUBLIC_IP_VPN_INTERFACE is set and up. It allows to check quicker if the VPN is up or if the public IP is already a registered ip.

### CODE REVIEW

**Comment by bhilburn** [comment]
Hey @lemaral - Thanks so much for submitting this PR! I apologize for the long delay in my response. It usually isn't quite so slow around here!

I really like this PR. It's a great usability improvement, and I think it is more intuitive than the current behavior. A couple of things:

1) I changed the merge target to `next`. Since this isn't *really* a bug fix and changes existing behavior, I'd like to merge it into `next` rather than `master.
2) This branch is currently failing unit testing, almost certainly because it changed the output of a segment. Are you up for updating the unit test to reflect your changes?

Thanks again for submitting this. I'm really excited to get this merged for our next release!

**Comment by lemaral** [comment]
Hi @bhilburn
Sure, I will check what's going on with unit testing, over the week-end hopefully :)
Thanks for this project, which I'm using all day long !

File: powerlevel9k.zsh-theme [hunk 1]
```diff
@@ -555,8 +555,16 @@ prompt_public_ip() {
...
+    icon='PUBLIC_IP_ICON'
+    # Check VPN is on if VPN interface is set
+    if [[ -n $POWERLEVEL9K_PUBLIC_IP_VPN_INTERFACE ]]; then
+      for vpn_iface in $(/sbin/ifconfig | grep -e ^"$POWERLEVEL9K_PUBLIC_IP_VPN_INTERFACE" | cut -d":" -f1)
+      do
+        icon='VPN_ICON'
+        break
+      done
+    fi
+    $1_prompt_segment "$0" "$2" "$DEFAULT_COLOR" "$DEFAULT_COLOR_INVERTED" "${public_ip}" "$icon"  fi
```
Comment (tippl) [inline powerlevel9k.zsh-theme]: This part is probalby where it causes unit test fails, as the tests which fail do not test this segment.
`fi` needs to be on a new line or be separated by a semicolon

Comment (lemaral) [reply powerlevel9k.zsh-theme]: Thanks @tippl for spotting that.
Indeed there is a missing line feed, which does not appear in my git diff.

**Comment by Artistan** [comment]
Nice!

**Comment by bhilburn** [comment]
@lemaral - Everything looks great! Thanks so much for putting together a new feature and upstreaming it! I'm excited to have this in the next release.

Thanks for a great PR, and welcome to P9k! `=)`

**Pull request merged**